crates/proof-of-sql/src/base/database/test_accessor.rs

```diff
@@ -34,7 +37,7 @@ pub trait TestAccessor<C: Commitment>:
 #[derive(Clone, Default)]
 /// A test accessor that leaves all of the required methods except `new` `unimplemented!()`.
 pub struct UnimplementedTestAccessor;
-impl TestAccessor<RistrettoPoint> for UnimplementedTestAccessor {
+impl TestAccessor<NaiveCommitment> for UnimplementedTestAccessor {
```

Comment:
```suggestion
impl<C: Commitment> TestAccessor<C> for UnimplementedTestAccessor {
```

Reply:
If you make these implementations generic, you don't need the `UnimplementedInnerProductTestAccessor`.